doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -247,6 +247,129 @@ And that's it! You are ready to run workloads on the cluster and check if the
 QoS requirements are being met.
 
 
+Switching Between Built-in and Custom Profiles
+==============================================
+
+There may be situations requiring switching from a built-in profile to the
+*custom* profile and vice-versa. The following sections outline the steps to
+achieve the same.
+
+Steps to Switch From a Built-in to the Custom Profile
+-----------------------------------------------------
+
+The following command can be used to switch to the *custom* profile:
+
+  .. prompt:: bash #
+
+    ceph config set osd[.N] osd_mclock_profile custom
+
+For example, to change the profile to *custom* on the OSDs, the following
```

Comment:
I think you mean "all the OSDs" here. I am wondering if we should only suggest changing the profile on all OSDs, not per OSD. I don't think we test mixed profiles, right?

Reply:
Yes, we haven't tested mixed profiles. I agree that we should suggest changing profiles on all OSDs and have changed the text and commands to reflect this.